spec/controller/open_library_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require_relative '../support/shared_context'
+
+RSpec.describe Api::V1::OpenLibraryController, type: :request do
+  describe 'GET book info' do
+    describe 'with valid isbn' do
+      before { get '/api/v1/open_library/0385472579' }
```

Comment:
What difference would there be between "before doing" and "before {}"?
Because placing it with before offends rubocop

![Captura de Pantalla 2021-06-01 a la(s) 17 44 00](https://user-images.githubusercontent.com/52846611/120388130-1d5dc400-c301-11eb-90aa-8549ca42a9ce.png)

![Captura de Pantalla 2021-06-01 a la(s) 17 41 15](https://user-images.githubusercontent.com/52846611/120387919-d7a0fb80-c300-11eb-9e5b-536bea6dbba3.png)

Reply:
fixed by adding this to travis config

`Metrics/BlockLength:
  Exclude:
    - 'spec/**/*.rb'`

Reply:
work same, only change the styling